plugins/Marketplace/resources/qml/OnboardBanner.qml

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2021 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.1
+
+import UM 1.6 as UM
+import Cura 1.6 as Cura
+
+// Onboarding banner.
+Rectangle
+{
+    Layout.preferredHeight: childrenRect.height + 2 * UM.Theme.getSize("default_margin").height
+    anchors
+    {
+        margins: UM.Theme.getSize("default_margin").width
+        left: parent.left
+        right: parent.right
+        top: parent.top
```

Comment:
This doesn't need to be anchored to the top, it's in a column layout. Because it's in a column layout, you're not allowed to anchor anything to the bottom or the top!

Reply:
https://github.com/Ultimaker/Cura/pull/10948/commits/c136deb430b1a0f9a82195eea28e6fbc0bcf2c3e